config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml

```diff
@@ -603,6 +603,31 @@ presubmits:
             cpu: 1
             memory: "2Gi"
 
+  - name: pull-perf-tests-verify-all-python
+    decorate: true
+    always_run: true
+    optional: true
```

Comment:
Please add a TODO to remove this once we prove it's stable.

Reply:
Done.